### DISCUSSION TITLE
[16] De-duplicate logs

### DISCUSSION THREAD

**Discussion by greatlypoint** [Security-Onion-Solutions/securityonion#1692] [Unsupported Versions] [answered]
How would I prevent duplicate logs going into Security Onion? We can't prevent duplicate logs on the upstream syslogs server... any ideas would be appreciated.

**Comment by dougburks** [comment, marked as answer]
Since you prefixed the subject with `[16]`, I assume you're referring to Security Onion 16.04.

Security Onion 16.04 questions should continue to go to the old Security Onion 16.04 Google Group as described here:
https://blog.securityonion.net/2020/10/community-support-forum-changes-for.html

Thanks!
